src/actions/plugins.js

```diff
@@ -6,11 +6,14 @@ import {bindClientFunc} from './helpers';
 
 import {PluginTypes} from 'action_types';
 
-export function getMarketplacePlugins() {
+export function getMarketplacePlugins(searchTerm) {
```

Comment:
Not sure what pattern we've converged on for actions, but would it make sense to destructure with a default here in anticipation of future parameters? 
```suggestion
export function getMarketplacePlugins({searchTerm: ''}) {
```

Reply:
Most actions in this repo pass the parameters separately so I kept that convention but I did add the default there to be cleaner.